src/Microsoft.VisualStudio.Composition/Configuration/Formatters/MetadataDictionaryFormatter.cs

```diff
@@ -12,16 +12,22 @@ namespace Microsoft.VisualStudio.Composition.Formatter;
 using Microsoft.VisualStudio.Composition.Reflection;
 using static Microsoft.VisualStudio.Composition.LazyMetadataWrapper;
 
+#pragma warning disable RS0016 // Add public types and members to the declared API
```

Comment:
We should not be hiding public API from the analyzer. Either we add it to the PublicAPI file or (in this case) we should make this internal. If making it internal breaks something, let me know and we'll find a way to fix it.